libraries/cmlib/cmlib_defs.mtlx

```diff
@@ -87,4 +87,24 @@
     <output name="out" type="color4" />
   </nodedef>
 
+  <nodedef name="ND_srgb_displayp3_to_lin_rec709_color3" node="srgb_displayp3_to_lin_rec709" nodegroup="colortransform">
+    <input name="in" type="color3" />
```

Comment:
Hi @dgovil . 

Minor thing: I have a fix so that default values are defined on the existing CM nodes [here](https://github.com/AcademySoftwareFoundation/MaterialX/pull/1369). Can you add in default values on the inputs here as well. I also added a cm node test file but can be patched later. Thanks.

Reply:
Done!